packages/common/src/ceramic-api.ts

```diff
@@ -54,6 +54,37 @@ export interface CeramicSigner extends CeramicCommon {
   [index: string]: any // allow arbitrary properties
 }
 
+/**
+ * Describes Ceramic Admin API functionality
+ */
+export interface AdminApi {
+  /**
+   * List indexed model streams
+   */
+  getModelsFromIndex(): Promise<Array<Stream>>
```

Comment:
That's a great call and something I wasn't sure about, thanks @stbrody !

Currently, if I understand correctly, `Model` is defined in `packages/stream-model` which depends on `packages/common` (where `CeramicAPI` is defined. So if I want to make the signatures more explicit, like you suggested, I either need to move the definition of `Model` to `packages/common` or do I deeper refactoring?

WDYT? Is there a better way of doing this?

Reply:
this would be a good question for @PaulLeCam or @ukstv who understand Typescript better than I do.  I believe you should be able to have the CeramicApi interface work with Stream but have the actual `Ceramic` and `CeramicClient` implementations of that interface specify that this actually a Model.

I'm also not totally clear on the difference between this interface specifying that the return type is `Stream` vs that it is `T extends Stream`